feature-libs/organization/account-summary/components/details/account-summary-mock-data.ts

```diff
@@ -10,11 +10,18 @@ import {
   DocumentStatus,
 } from '@spartacus/organization/account-summary/root';
 
+const PURCHASE_ORDER = 'Purchase Order';
+const JAN_24 = '2022-01-24';
+const JAN_26 = '2022-01-26';
+const FEB_25 = '2022-02-25';
```

Comment:
I am thinking if wouldn't be better to create an array of dates, instead of separate variables.